menu/tests/tests_menu.py

```diff
@@ -0,0 +1,231 @@
+from django.urls import reverse
+
+from rest_framework import status
+
+
+class TestValidateMenu:
+
+    def test_menu_exist(self, api_client, custom_location, custom_product):
+        response = api_client.get(
+            reverse('display_menu', args=[custom_location.code]), format='json')
+        assert response.status_code == status.HTTP_200_OK
+        response_json = response.json()['results'][0]
+        assert response_json
+        assert response_json['id'] == custom_product.id
+        assert response_json['name'] == custom_product.name
+        assert response_json['description'] == custom_product.description
+        assert response_json['volume'] == int(custom_product.volume)
+        assert response_json['measure'] == custom_product.measure
+        assert custom_product.cost in response_json['cost']
+        assert response_json['logo'] is not None
```

Comment:
think about separate

Reply:
change to 
        assert response_json['count'] == 1

Reply:
change to
len(response_json) == 10